docs/data/data-grid/recipes-row-grouping/RowGroupingSortByChildRows.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  gridFilteredDescendantCountLookupSelector,
+  useGridApiRef,
+  GRID_ROW_GROUPING_SINGLE_GROUPING_FIELD,
+} from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+export default function RowGroupingSortByChildRows() {
+  const data = useMovieData();
+  const apiRef = useGridApiRef();
+
+  return (
+    <div style={{ height: 400, width: '100%' }}>
+      <DataGridPremium
+        {...data}
+        apiRef={apiRef}
+        initialState={{
+          rowGrouping: {
+            model: ['company', 'cinematicUniverse'],
+          },
+          sorting: {
+            sortModel: [
+              { field: GRID_ROW_GROUPING_SINGLE_GROUPING_FIELD, sort: 'desc' },
+            ],
+          },
+        }}
+        groupingColDef={{
```

Comment:
Should we stabilize the reference to this prop?
1. [`getGroupingColDefs`](https://github.com/mui/mui-x/blob/next/packages/grid/x-data-grid-premium/src/hooks/features/rowGrouping/useGridRowGroupingPreProcessors.ts#L53) might recompute on each render which may yield some rendering side effects.
2. In docs, we [suggest](https://next.mui.com/x/react-data-grid/faq/#how-to-pass-props-to-the-data-grid-for-better-performance) to keep stable references for all non-primitive props, so it'd be better to follow the same in our demos.

Reply:
Done 👍🏻